src/index.html

```diff
@@ -4,20 +4,28 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>ToDoList</title>
+  <link rel="stylesheet" href="./style.css">
 </head>
 <body>
   <main>
     <div class="div-heading">
       <h1>Today's To Do</h1>
     </div>
 
-    <div class="list-adding"></div>
+    <div class="list-adding">
+      <input type="text" placeholder="Add to your list..." id="add-new-list">
+      <button type="button" class="addbutton">
+        <img src="./enter.png" alt="addicon" class="addicon">
+      </button>
+    </div>
```

Comment:
- [ ] Good job so far, :clap: However, Please ensure that you check your class `list-adding` and id `add-new-list` as your form is showing two input fields with only one working at the moment, Kindly remove the one input field. :slightly_smiling_face: 

![Screenshot](https://github.com/mj-e-boc/To-Do-List/assets/43528594/5a79e5c7-45b1-4516-b9de-8344c47fe898)